tests/test_install_script.py

```diff
@@ -110,24 +110,21 @@ def verify_yt_installation(binary_yt):
             continue
         if dep == "git":
             git_path = os.sep.join([yt_dir, "bin", "git"])
-            call_unix_command("{} --version".format(git_path))
+            call_unix_command(f"{git_path} --version")
         if dep in DEPENDENCY_IMPORT_TESTS:
             cmd = "{} -c '{}'"
             if dep == "rockstar":
                 # compiling rockstar is broken on newer MacOS releases
                 if platform.mac_ver()[0].startswith(("10.12", "10.13")):
                     continue
                 cmd = (
-                    "LD_LIBRARY_PATH={} ".format(
-                        os.sep.join([os.curdir, yt_dir, "lib"])
-                    )
-                    + cmd
+                    f"LD_LIBRARY_PATH={os.sep.join([os.curdir, yt_dir, 'lib'])} " + cmd
```

Comment:
I know this isn't your code, but it could be replaced by
```suggestion
                    f"LD_LIBRARY_PATH={os.path.join(os.curdir, yt_dir, 'lib')} " + cmd
```

Reply:
I'd rather keep this PR as pure as possible. I replicated those automated changes in #3020